features/step_definitions/bootstrap_steps.rb

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2010-2017 SUSE-LINUX
+# Licensed under the terms of the MIT license.
+
+And (/^I create the bootstrap-repo for "([^"]*)" on the "([^"]*)"$/) do |target, server|
```

Comment:
here i would remove the server as a target
```And (/^I create the bootstrap-repo for "([^"]*)" on the server$/) do |target|```

because the $server.run , see last comment